core/vibes/soul/form/card-radio-group/index.tsx

```diff
@@ -13,55 +13,102 @@ interface Option {
   disabled?: boolean;
 }
 
+/**
```

Comment:
Wondering why this comment doesn't need disable jsdoc lint rule like other components 🤔